### PR TITLE
make universe settings working in the new ui

### PR DESCRIPTION
**Pull request by daveol**
The tab for universe settings in the new ui now actually works.

### CODE REVIEW

File: olad/www/new/views/universe-settings.html [hunk 1]
```diff
@@ -75,6 +113,29 @@
...
+    <td>
+     <div class="checkbox-tr">
+      <input type="checkbox" ng-model="Data.Add[$index]" ng-true-value="'{{port.id}}'" ng-false-value="''">
+     </div>
+    </td>
+    <td>{{port.device}}</td>
+    <td>{{port.description}}</td>
+    <td>
+     <div ng-if="port.priority.priority_capability">
+      <div ng-if="port.priority.priority_capability == 'full'">
```
Comment (peternewman) [inline olad/www/new/views/universe-settings.html]: Should this be ===, or does that not work with templates?

File: olad/www/new/views/universe-settings.html [hunk 2]
```diff
@@ -75,6 +113,29 @@
...
+       <select class="form-control priority" ng-model="port.priority.current_mode">
+        <option value="inherit">Inherit</option>
+        <option value="static">Static</option>
+       </select>
+      </div>
+      <div class="static-fake" ng-if="port.priority.priority_capability == 'static'">static</div>
```
Comment (peternewman) [inline olad/www/new/views/universe-settings.html]: Again?

File: olad/www/new/views/universe-settings.html [hunk 3]
```diff
@@ -75,6 +113,29 @@
...
+     </div>
+    </td>
+    <td>
+     <input class="form-control priority" ng-if="port.priority.current_mode" ng-model="port.priority.value" type="number" ng-disabled="port.priority.current_mode == 'inherit'"/>
```
Comment (peternewman) [inline olad/www/new/views/universe-settings.html]: Again?

**Comment by peternewman** [comment]
Looks good @daveol , just some minor comments.

**Comment by peternewman** [comment]
LGTM.

**Comment by peternewman** [comment]
@nomis52 do you want to take a look? I'm happy with it.

**Comment by nomis52** [comment]
Yeah, I had a partial issue filed and then my browser quit :(. I'll try to get to it today.

**Pull request merged**